tests/functional/adapter/concurrent_transaction/fixtures.py

```diff
@@ -0,0 +1,60 @@
+create_udfs_sql = """
+{% macro create_udfs() %}
+
+CREATE OR REPLACE FUNCTION {{ target.schema }}.f_sleep (x float)
+RETURNS bool IMMUTABLE
+AS
+$$
+  from time import sleep
+  sleep(x)
+  return True
+$$ LANGUAGE plpythonu;
+
+{% endmacro %}
+""".lstrip()
+
+#
+# incremental materialization
+#
+incremental_model_sql = """
+{{ config(materialized='incremental', unique_key='id') }}
+
+-- incremental model
+select 1 as id
+
+{% if is_incremental() %}
+    where TRUE
+{% endif %}
+""".lstrip()
+
+incremental_view_sql = """
+select * from {{ ref('model_1') }}
```

Comment:
The three views are the same model. Do we need to have all three? I think the idea is that this model can be built regardless of the materialization of the dependent model. So we just need to define it once and then swap out the dependent model for the three tests. That would also make the block comments unnecessary.